index.js

```diff
@@ -52,19 +55,17 @@ module.exports = {
     'declaration-bang-space-before': 'always',
     'declaration-colon-space-after': null,
     'declaration-colon-space-before': 'never',
-    // 'declaration-no-important': [true, {'severity': 'warning'}],
 
     // Declaration block
     'declaration-block-no-duplicate-properties': true,
-    'declaration-block-no-ignored-properties': [true, {'severity': 'warning'}],
```

Comment:
I _guess_ that's what csstree validator is all about, but it is notoriously badly documented. I can look into it later.